packages/office-ui-fabric-react/src/components/Nav/Nav.types.ts

```diff
@@ -11,6 +11,14 @@ export interface INav {
    * all groups of the Nav must have populated key properties.
    */
   selectedKey: string | undefined;
+
+  /**
+   * Sets focus to the first tabbable item in the zone.
+   * @param forceIntoFirstElement - If true, focus will be forced into the first element, even
+   * if focus is already in the focus zone.
+   * @returns True if focus could be set to an active element, false if no operation was taken.
+   */
+  focus?: (forceIntoFirstElement?: boolean) => boolean;
```

Comment:
Where is the API file change? Did you run `yarn update-api`?

Reply:
This is the version 5 branch which doesn't have API extractor